### DISCUSSION TITLE
Update to Our Privacy Policy & Apology

### DISCUSSION THREAD

**Discussion by AudacityTeam** [audacity/audacity#1353] [General]
Today, we are introducing a revised privacy policy to address the concerns raised with the previous privacy policy published earlier in July. The original policy, which was drafted in anticipation of the release of Audacity 3.0.3, received a large amount of media attention due to the inclusion of a few key provisions that appeared to some to violate promises we had previously made. We want to address the issues that were raised, give assurances about our intentions, and provide verifiable proof of what information is actually sent from Audacity. The new privacy policy uses clearer language that we hope will explain our intentions more accurately this time. We are deeply sorry for the significant lapse in communication caused by the original privacy policy document.

**Updated privacy policy**
https://www.audacityteam.org/about/desktop-privacy-notice/

### **Summary of main changes**

- Phrasing has been adjusted to remove ambiguity or aid in transparency, in particular that we do **not** collect any **additional** information for law enforcement or any other purpose
- We have explained the purpose of the two networking features, error reporting and update checking
- We have removed the provision that discourages children under 13 years old from using Audacity
- We have taken steps to ensure that we never store a full IP address (we now truncate it before hashing or discard it entirely) and have reflected this change in the privacy policy document
- We have made some changes to how we process error reports to ensure that we never store any potentially identifiable information

### **Unclear phrasing**

We drafted the original privacy policy as a legal text. We appreciate that for our community, as well as our users, much of phrasing in the policy produced more questions than answers. From now on we will provide context for changes we make to the policy in a user friendly way. Part of the problem with the original privacy policy is terminology. There are terms we are legally required to use in order to ensure compliance with the GDPR and the CCPA. The best example is the catchall phrase ‘**personal information**’, a non-specific term that understandably raises concerns for regular readers. While that term must still appear in the new privacy policy, we have tried to be a lot more specific about the actual information we are referring to wherever possible.

To provide an example of where use the term 'personal information' is unavoidable, in section 5.1 there is a line under **Data security** that states '_We use appropriate technical and organisational measures to protect the **personal information** that we **collect and process**_'. Please note that an IP address alone counts as 'personal information', and the steps we take to anonymise it count as 'processing'. When you see the line in 4.1: '**_We do not store or share any personal information_**.', this refers to the fact that, while we see (i.e. 'collect') the full IP address, we do not store it (more about this below). 

The most unclear and damaging part of the original document stated that we collect personal information “**…necessary for law enforcement, litigation and authorities' requests (if any)**”. This was interpreted to mean that we intended to collect and store unspecified additional information on top of the basic system information mentioned elsewhere in the privacy policy. This is not true, as could be seen through inspection of the source code and network analysis of the release binaries. However, we agree that the wording used in the old privacy policy made it sound like it _might_ be true. We have now changed the wording to remove this source of confusion. To be clear, any organisation, if ordered by the court, is required to cooperate with an investigation, and doing otherwise is considered to be an obstruction of justice. These are not the rules we create, these are the requirements we must follow. However, we would only be able to provide the specific information mentioned in the privacy policy (outlined below) and nothing more. In addition, the steps we have taken to anonymise all stored data means that it would be of extremely limited use to anyone.

It is verifiably untrue that we hid the exact data being collected. As noted by journalists who investigated the issue, Audacity is free and open source software, and an inspection of its source code shows that the data it shares is extremely limited. This was already the case at the time the original privacy policy was published and has not been changed since, as can be confirmed from the commit history in our repository and in others.

### **Update Checking**

We notify users when there is a new version of Audacity available. This requires a network connection, and shares your IP address, OS, and the Audacity version. This is the only information that the program shares by default and it can be disabled in Preferences at any time, as we promised in #889.

Regarding IP addresses, when Audacity checks for updates, we use the IP address to determine the country the user connected from but nothing more detailed than that. We have set up our systems in such a way that IP addresses are immediately anonymised (technically speaking, we truncate the IP address to 3 bytes out of 4 and then hash it). This makes it impossible for us to identify the user or pinpoint their exact location, yet it should still be enough to help detect and mitigate spam update requests or possible DoS attacks on our servers.

After anonymisation, the data we store provides basic statistics about the daily number of users we have per country, operating system (Windows, Mac or Linux), and Audacity version. We would be able to tell on a given day how many people in the USA used a particular version of Audacity on a particular operating system, but we would be unable to dig any deeper than that. These statistics are extremely useful for planning and development. For example, it will tell us how many people use Audacity on old operating systems, which will help us to decide when it is reasonable to drop support for those old operating systems in favour of newer technologies (e.g. macOS version 10.9 is required for minimal C++17 support).

### **Error reporting**

Besides the check for updates feature, the only other feature in the upcoming Audacity release that requires use of a network connection is error reporting. If an application error occurs, a popup appears asking you whether you want to send us the details of that error, which you can review before sending. Since the previous privacy policy was released, we have taken additional steps on the server to ensure that all potentially identifiable information is filtered out of report data before storage. We were already filtering out file paths from non-fatal error reports within Audacity, but now we have configured our Sentry server to filter file paths from crash reports too. We do not store an IP address for any kind of error report.

In terms of the data sent by Audacity, we have only made one change since publication of the previous privacy policy: we have removed the ability for users to send a comment as part of an error report. This was done to prevent users typing personal information in the comments field, which we have no reliable way to filter. With the rest of the report's technical data anonymised, the comments field was the last way that we might have obtained identifiable information. With the comments field gone, we were able to remove all references to sharing of personal information in the updated privacy policy, because after anonymisation (which occurs more or less instantly as data arrives on the server) there is no personal information for us to share.

### **Building from source**

When you build Audacity from source code all network features are excluded by default. We expect Audacity builds available via Linux distributions to be network-free unless the maintainers of said packages specifically want to include networking features.

### **Revision of age restrictions**

The old privacy policy contained a provision that discouraged children under 13 years old from using the program. After extensive further consultation with our lawyers, we have determined that this provision is unnecessary given the actual mechanics of data transmission and storage. The provision had been included out of an abundance of caution, but in the end turned out not to be required. We sincerely apologise for including this provision in the original privacy policy, and we are pleased to confirm that Audacity will remain freely available to users of all ages.

### **Conclusion**

We accept that the provision about under 13s was unnecessary and we accept certain other clauses in the old privacy policy were worded in a way that begged for misinterpretation. However, we hope we have sufficiently demonstrated that very little data is actually collected and that you have full control over whether or not any data is sent, and that this was already the case before the story received any media attention. We'd like to repeat our apology for the confusion our old privacy policy has caused. We are now taking steps to improve our processes for releasing any information related to Audacity in the future to ensure that users are appropriately informed. We understand that some people may have questions concerning the revised Privacy Policy, which we will do our best to answer here.

**Comment by SteveDaulton** [comment]
Very pleased to see these changes, and as  someone that  has worked for many years in education, I'm particularly pleased to see that the item about under 13's  has been scrapped.

**Reply by blooalien** [reply]
> I'm particularly pleased to see that the item about under 13's has been scrapped.

That change made me especially happy, too.  Anything that risks depriving children of access to educational uses of good open source software resources can only be a bad thing.

**Reply by edo0** [reply]
While that has been changed, unfortunately Audacity is still captive of Muse Group

https://twitter.com/marcan42/status/1417085393762099200

For everyone looking into a fork, [Tenacity](https://github.com/tenacityteam/tenacity) is the most promising one

**Reply by blooalien** [reply]
> For everyone looking into a fork, [Tenacity](https://github.com/tenacityteam/tenacity) is the most promising one

Oh, for sure Audacity still needs a fork.  This latest development absolutely does not change that fact in the slightest.

**Comment by petersampsonaudacity** [comment]
It's good that there are clear links to the revised PP in the app itself
a) from the initial update check dialog on first launch of 3.0.3
b) from the Help>About Audacity>Legal

**Comment by CGPipeliner** [comment]
Thanks a lot, I really appreciate this update!

**Comment by AudacityTeam** [comment]
We have already deleted a comment from a user who is introducing a topic completely irrelevant to this discussion and not actionable by us. We have written a moderation policy to make our criteria for comment deletion clear. We apologise for taking this step, which we feel is necessary.

https://github.com/audacity/audacity/discussions/1354

**Reply by AudacityTeam** [reply]
Although we want to make sure we own our mistakes - and we do not expect any relief from criticism by doing so - we also want to make it clear that the vast majority of the coverage on this issue was based on something that was not true. We never at any point sought to collect unspecified information for any purpose and that is verifiably the case.

When people come to you and asking what the truth of the matter is, you should send them to this page so they can see for themselves.

**Comment by ann4belle** [comment]
The fact remains that not only did you try to force this through, taking **two and a half weeks** to make some common-sense changes to a policy that didn't even need to be updated in the first place, but that this is the second time you've tried to pull a stunt like this. You need to either stop intentionally trying to start fights with the open source community, or find another maintainer that understands how things work here.

Bottom line, if you had just started a conversation about your concerns over the existing privacy policy vs. what data you wanted to collect instead of just trying to cram an updated policy down our throats by effectively saying "this is how it is now, whether you like it or not", *none* of this would've happened.

**Reply by Tantacrul** [reply]
So, I think it's important to point out that we did not actually try to force through any feature that wasn't previously discussed in #889. This is one of our key points in the above statement: the interpretation of the privacy policy did not correspond with fact. This is (of course) a fault with the badly written privacy policy, not the user.

**Reply by shoogle** [reply]
It did take a long time for us to update the privacy policy, but we released a [statement](https://github.com/audacity/audacity/discussions/1225#discussion-3446544) on the first working day after the original privacy policy was published saying that we were working on a new one. The statement made clear the sort of things we were looking to improve in the new privacy policy, and since that day we have been working flat-out with our legal, technical, and communication teams to get the language exactly right in the legal document.

**Reply by wchristian** [reply]
@ann4belle 
> not only did you try to force **this** through

@Tantacrul 
> we did not actually try to force through any **feature**

This is not how good faith works, Tantacrul.

At best this is a really silly and hasty mistake from your end. At worst this is an attempt from you to redirect the conversation by maliciously rephrasing the original words.

**Reply by Tantacrul** [reply]
The only rephrasing that occurred turned inaccuracies in the PP to accuracies. I'll repeat: nothing was forced through that was not already explained in #889.

**Reply by wchristian** [reply]
The rephrasing happened in the quoted sentences. I am baffled why you would decide to act as if i was talking about changes to the PP when i used that word. If you honestly didn't understand me, fine, but please understand this looks sketch as fuck.

(I'd also like to note she said *try* and you responding as if she said you *did* force something true is not a good look either.)

**Comment by petersampsonaudacity** [comment]
I have been doing 3.0.3 Release Candidate testing and all seems good:

a) error and crash reporting are one-off opt-in each time you get such a dialog, 

b) the update checking dialog that you get on initial launch of 3.0.3 enables you to turn off update checking in Application prefs BEFORE you hit the OK button - and thus no data is sent.

**Comment by SanJacobs** [comment]
Cool, now when are you removing the CLA?

**Reply by roland-tb303** [reply]
I'm wondering that too. CLA's sketchy af. Although, buyer companies being buyer companies, I don't think they'll do much.

**Reply by SteveDaulton** [reply]
Speaking personally, there's some key features of the CLA that make it acceptable for me.
1. It is a **non-exclusive** license. You are still free to use your contributed code anywhere you like under any license terms that you want.
2. Your contributed code will be free for anyone to use forever under the license terms that applied at the time of your contribution. You are only granting permission for Audacity to release future versions under different license terms, but the terms of the original code are irrevocable. The obvious example is that under the CLA, Audacity can be released under GPL v3 (which is compatible with VST3*), but your original contribution may still  be used under the current GPL v2 license.
3. The contributor retain copyright of their contribution. In other words, the CLA grants "permission to use" and is not a transfer of ownership.

Having looked at a number of other open source CLA agreements, this one appears to me to be one of the least restrictive - basically  allowing the contributor to do what they like with their code, and retain ownership (copyright) of their work.

I'm not trying to persuade anyone, in fact I strongly believe that every contributor has the right to make their own decision about whether they want to contribute to Audacity or not. I first contributed to Audacity about 15 years ago because I used Audacity and found it had a bug that annoyed me, so I figured out a fix and contributed it - personally I'm very happy that millions of other Audacity users have benefited from that fix.

* As I'm a Linux user, I'm unlikely to benefit from VST3 support, but I know many Audacity users on Windows that would love to  see Audacity have better support for VST plug-ins, but Steinberg's current license is only compatible with GPL v3, and not GPL v2 https://developer.steinberg.help/display/VST/VST+3+Licensing

**Reply by prokoudine** [reply]
@SteveDaulton 

As a matter of fact, you have pretty good chances to benefit from VST3 support. VST3 is quickly becoming a rival to LV2 on Linux as well. There's a bunch of free/libre virtual instruments available as VST3 (sometimes before they are available as LV2) and there are some VST3 audio plugins as well (I believe OverTone DSP will also move from VST2 to VST3 for Linux soon).

**Reply by SteveDaulton** [reply]
@prokoudine
 Nice, I didn't know that.

**Reply by cjjd** [reply]
> 2. Your contributed code will be free for anyone to use forever under the license terms that applied at the time of your contribution. You are only granting permission for Audacity to release future versions under different license terms, but the terms of the original code are irrevocable. The obvious example is that under the CLA, Audacity can be released under GPL v3 (which is compatible with VST3*), but your original contribution may still  be used under the current GPL v2 license.

You are not the first person that has said that. Can you show me in the CLA where it says this? Allowing them to use contributions "in any way" seems to make this not true. There doesn't appear (to me) to be any provision in the CLA that audacity will continue to release a free and/or open source version.

**Reply by SteveDaulton** [reply]
The rights that I referred to are written into the GPL license agreement, which is legally binding.

Let me ask  you a couple of questions:
1. If you develop an app and license it under GPL v2, are you obliged to continue to release new versions of the app under the same license forever?
My understanding of the GPL license is that there is no such obligation.
2. If  you develop and release an app under the terms of GPL v2, can you _retrospectively_ change the license terms of the app that you have released?
My understanding of the GLP license is that you cannot retrospectively change the license terms.

Here's the GPL v2 license: https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
What's your interpretation?

**Reply by SanJacobs** [reply]
Lucky for us, it is entirely possible to relicense to GPLv2+ without enforcing a CLA.

**Reply by cjjd** [reply]
>If you develop an app and license it under GPL v2, are you obliged to continue to release new versions of the app under the same license forever?
>My understanding of the GPL license is that there is no such obligation.


This is my understanding as well. If the GPLV2 had this, the issue would be moot. I also understand that this will never be included as it would limit a lot of the rights of the copyright holder. There could be no dual-licensing, etc. 

>If you develop and release an app under the terms of GPL v2, can you retrospectively change the license terms of the app that you have released?
>My understanding of the GLP license is that you cannot retrospectively change the license terms.


This seems to be the conventional wisdom, but I don't see explicit language in the license to indicate that it is perpetual. That being said, the practicality of rescinding an open source license with the code being out there already may be an exercise in futility. You'll of course notice that Muse also seems to think revoking is an issue as they explicitly include the words perpetual and irrevocable in the CLA. 

Right now, development of audacity is done publicly on github. There is nothing preventing muse from making the repository private and releasing source when binary releases happen. There is also nothing preventing them from deleting the repo entirely and only releasing proprietary versions (once they are done stripping out the GPL only libraries). I understand that the community can then fork and do whatever, but the issue I believe a lot of people have is that Muse hasn't indicated anywhere that they will release contributions under an open source license. Muse has not made a binding commitment to open source in any formal way. 

So again, where does Muse, who the contributors are contractually giving the code to, say that they will release it under the open source license? What about when they develop proprietary derivatives? What is Muse actually committing to?

**Comment by ThatRandomGamer** [comment]
I am sorry but all trust has been lost. I loved Audacity and promoted it whenever possible, now, I'm sticking with my old 2.x build and never updating. Maybe trust will build up in the future but it's not there right now after all the shit that went down the past 2-3 months

**Reply by AudacityTeam** [reply]
Understandable. We are aware we have a mountain to climb and we're not going to allow another unnecessary fiasco like this happen again.

**Reply by wchristian** [reply]
The only mountain you have to climb at this point would be to return audacity to being a project that is not owned by Muse Group.

**Reply by ProfessorCode212** [reply]
> The only mountain you have to climb at this point would be to return audacity to being a project that is not owned by Muse Group.

That's not fair. I am feeling a little skeptical too but that's a bit of a _stretch_. 

> Understandable. We are aware we have a mountain to climb and we're not going to allow another unnecessary fiasco like this happen again.


@AudacityTeam, I do sincerely hope that you live up to that promise.

**Comment by nimishgautam** [comment]
I can appreciate some of the problems around open-source as a business model, coupled with an entitled user culture can make for a very discouraging environment to work in. In that spirit, I just want to say thank you :) 

There are many of us that have been using audacity for **years** and would be lost without it. I know about a zillion podcasters who wouldn't be able to do what they do without the software you produce. The question us tech-minded folk get asked in these kinds of communities is "should we keep using Audacity?" As of now, I've been saying 'yes' and intend to continue saying that.

This is a very jarring way to suddenly learn your team needs better communication and transparency with users, but I think you've taken some strong steps to show you're listening, and I hope you continue to grow in that regard.  In the meantime, keep up with the amazing work, and thanks again :)

**Reply by mtwb47** [reply]
I hope that they've learned. But it's going to take time for that to be proven to the community. I too will continue to use Audacity, but it wouldn't take a lot -- another misstep -- for me to find something else.

**Comment by ekianjo** [comment]
> We notify users when there is a new version of Audacity available. This requires a network connection, and shares your IP address, OS, and the Audacity version. This is the only information that the program shares by default and it can be disabled in Preferences at any time, as we promised in #889.

That should be disabled by default. On Linux, I will know if an update is available via my repositories. I want no direct connection done by default to a site I do not trust, and certainly not with my IP nor my OS information. That's also against the GDPR to send such information without asking the user prior agreement.

**Reply by prokoudine** [reply]
@ekianjo 

> That should be disabled by default. On Linux, I will know if an update is available via my repositories. 

That is precisely the case with Audacity. When you build it from source code, all networking features are disabled by default. The maintainer of the 'audacity' package in your distro of choice would have to specifically enable networking features. The explanation that you are commenting on seems very clear on that.

**Reply by Tantacrul** [reply]
Just a little lower in our statement, there is a section titled 'Building from source' which I hope addresses your concern.

> **Building from source**

> When you build Audacity from source code all network features are excluded by default. We expect Audacity builds available via Linux distributions to be network-free unless the maintainers of said packages specifically want to include networking features.

edit: Also, as stated above, we **do** notify the user before making an update check or error report.

**Reply by Tantacrul** [reply]
Snap

**Reply by SteveDaulton** [reply]
> On Linux, I will know if an update is available via my repositories

In the case of a distribution's repository version, that is a matter for the package maintainers to decide. In keeping with normal open source practice, package maintainers are free to configure, build and package as they feel appropriate. Networking is disabled by default when building from source, so it will only be enabled in Linux distro packages if the package maintainers explicitly override the defaults and enable it. 

Personally I think it is  extremely unlikely that Linux maintainers would enable update checking, because, as you say, Linux has its own update mechanisms so there's no need, but the point is that it's the Linux package maintainers choice and NOT something imposed from "upstream".

For anyone that build from source, networking is disabled by default. If for any reason you want to enable networking, you must override  the defaults  and enable it in your configure options.

(apologies for the repetition - I'm slow at typing ;) )

**Reply by ron-wolf** [reply]
> @Tantacrul
> Snap
> <kbd>👎 2</kbd>

I wonder what your second comment did to deserve the dislikes? Are people just blindly going down the page and hating on you guys?

Mad respect for what you guys are doing, still engaging and compromising with what is in part an extremely toxic community (with poorly defined boundaries, as is common on the Internet). This is the very definition of turning the other cheek.

**Comment by Gregor-Davies** [comment]
What stops you guys from taking another u-turn in the future? Muse has already shown their true colours (Not good ones) when it comes to opensource projects.

**Reply by wchristian** [reply]
This is the crucial question.

When making a mistake it's not enough to just address the mistake itself, but also explain how you make sure to not repeat it, especially when the mistake causes distrust.

**Reply by AudacityTeam** [reply]
There is no U-turn here. We are releasing (in Audacity 3.0.3) what we said we'd release in #889, with the exception of a comments box, which we've removed from the error reporting feature. Outside of Audacity, the only other changes are the wording of the privacy policy and improvements to how we anonymize IP addresses.

**Reply by Gregor-Davies** [reply]
> There is no U-turn here. We are releasing (in Audacity 3.0.3) what we said we'd release in #889, with the exception of a comments box, which we've removed from the error reporting feature. Outside of Audacity, the only other changes are the wording of the privacy policy and improvements to how we anonymize IP addresses.

If you can make a u-turn on your privacy policy which affects thousands of users what stops you for instance changing other bigger things?

**Reply by different55** [reply]
> > There is no U-turn here. We are releasing (in Audacity 3.0.3) what we said we'd release in #889, with the exception of a comments box, which we've removed from the error reporting feature. Outside of Audacity, the only other changes are the wording of the privacy policy and improvements to how we anonymize IP addresses.
> 
> If you can make a u-turn on your privacy policy which affects thousands of users what stops you for instance changing other bigger things?

There was no U-turn. And this is the literal definition of a slippery slope.

**Reply by wchristian** [reply]
@AudacityTeam You guys keep doing this and i can't tell if you're ignorant or malicious. The claim is not, and was not made by any significant amount of people, that you did release something harmful. The claim is that you were *trying* to, and since #835 exists, you cannot deny that.

The question is how you will head something like that off before it even gets that far.

**Comment by VidiotGeek** [comment]
I say bravo! You messed up and you seem to have taken that mistake seriously and taken proper steps to remedy it. Let the folks who still have issues make their forks and change--whatever line of source code they can find that they still have a problem with, I guess? I'll take version 3.0.3. If there eventually is some major change that's worthy of this level of concern in the future, we can all piously jump ship then and congratulate one another on knowing it all along at that time.

**Comment by different55** [comment]
Please release an official statement on the xmader incident. The existing statement does not address the problematic nature of the employee's actions which amount to a veiled death threat. I've defended you on the audacity front but I cannot when it comes to Xmader.

**Reply by AudacityTeam** [reply]
We cannot as the Audacity team - who have nothing to do with the MuseScore.com catalog - comment on this here. When the issue is addressed, we will link to it.

**Reply by different55** [reply]
This is noted and appreciated.

**Reply by ron-wolf** [reply]
> which amount to a veiled death threat

Please tell me you understand how this is an extremely narrow and non-contextualized interpretation…

**Reply by different55** [reply]
> > which amount to a veiled death threat
> 
> Please tell me you understand how this is an extremely narrow and non-contextualized interpretation…

Would legitimately be happy to discuss this with you but I'm not sure this is the proper place to do it.

**Comment by CranialTyrant** [comment]
These are steps in the right direction, but there's one thing I take issue with that might seem minor: In the section "Revision of age restrictions," even though they removed the 13 and under age restriction from the privacy policy (which is good), they do not acknowledge that the change would have violated the GPL. They need to add that to this section.

Again, I'm happy Muse is taking steps in the correct direction and righting their wrongs, and if they keep it up, and do good by the community, I'll more than consider using Muse's Audacity. 
However, I must note that it will take time for trust to be regained, and if something else like this comes up, Muse would just be removing dirt from the hole that needed filled.

Edit: Also, the CLA is still a really big stain that needs cleaned.

**Comment by troysjanda** [comment]
Saving face in hopes of leveling off loss of userbase seem pathetic in my opinion. As a long time user I have lost my trust and sought out  another program to use. As a developer I see the usefulness of collecting telemetry but should always be opt in and should be available pre launch of the program to give the user the ability to opt-out before said telemetry even has a chance to be collected and transmitted, period...

**Comment by zearp** [comment]
Remove the network stack. This is an audio editor not a web browser. People who want to help fix bugs can copy/paste or email crash reports. There is no need to add a complete networking stack just to make it easier to send crash reports. This will also put at ease anyone worried about tracking/data collecting. Did any of the active forks remove the network stack completely?

**Reply by petersampsonaudacity** [reply]
>People who want to help fix bugs can copy/paste or email crash reports.

Sadly folk don't do that - very seldom in the dozen or so years I worked on Audacity QA and support (retired now) we never got such useful information from users when they reported bugs - just vague descriptions of the bug result and mostly no idea of the run-up scenarios ad no steps-to-reproduce.

**Reply by CranialTyrant** [reply]
Did any of the active forks remove the network stack completely?

I believe Tenacity and Sneedacity have.

**Reply by prokoudine** [reply]
> There is no need to add a complete networking stack just to make it easier to send crash reports.

I wish more programs would simplify generating backtraces and sending them to developers. Getting a user to run an app through gdb, replicate the issue, and copy/paste 'bt' output is difficult enough to justify this kind of a solution.

**Comment by geekredux** [comment]
Audacity was first released in 2000 and in the succeeding 21 years, despite its imperfections, grew to become the beloved and perhaps _de facto_ standard audio editor for countless amateurs around the world. 21 years of positive reputation, goodwill, and trust--and you wrecked it in a matter of weeks. 

Starting with #835 and proceeding to the the privacy policy addressed in this post, you made it clear where your priorities lie. When challenged on them by users here, you double-downed on why you wanted to do these things and why it should be okay before walking back on the grounds of "miscommunication", and then only after drawing the attention of media.

Sure, things as they stand now may be in a more acceptable state, but I have zero faith that Muse Group won't be trying again to make grabs for more user data than relevant or necessary for running an audio editor--only next time, you'll know to start turning up the water temperature very slowly instead of just tossing things into a boiling pot.


ETA: added trust to the list of positives in Audacity's 21 year history

**Comment by Albonycal** [comment]
Well.. 
that looks good
but I don't think audacity needs any telemetry..
Everyone knows what needs to be improved 
The **UI** It looks like it's from 90s

**Reply by kylxbn** [reply]
It seems that there never was telemetry. There is only update checking and error reporting, and both are disabled by default--you are asked if you want to enable them and if so, they will be enabled.
It's all a huge misunderstanding blown up by the media.

**Reply by geekredux** [reply]
More info on the telemetry issues in #835.

**Reply by kylxbn** [reply]
@geekredux Thank you for pointing that out! I did not know that. Still, as long as telemetry is not personally identifiable and only transmits vague data (for example, not audio samples and filenames) I think it would be beneficial for the developers to know what things to prioritize or change. In any case, it is disabled by default and will only be enabled if you agree to it, so I still stand by my opinion that this is just a huge misunderstanding blown up by media coverage. I don't deny the fact that the original privacy policy was very suspicious, though. But I'm glad thatthey are trying to fix it.

**Reply by LWinterberg** [reply]
UI improvements are coming soon! You can get a rough sketch/wireframe of things to come here: https://vimeo.com/562162689

**Reply by Albonycal** [reply]
hmm nice

**Comment by kylxbn** [comment]
Thank you very much for the clarification.

I have always loved Audacity ever since I had my first computer. I always use it for my personal hobby projects. Hearing about the overblown news from the media did concern me, but since I don't know the truth, I didn't know whether to believe or not. In the meanwhile, even though I'm a bit concerned, I kept on using Audacity with the hope that it's all just a big misunderstanding, or of it isn't, then I just hoped that the Audacity Team do not have evil plans when it comes to the personal data collected. And you now come with the clarification that it is indeed a big misunderstanding.

I am happy with the current network features, I use Audacity in 3 different OSes, and while I prefer Linux, the other two OSes will be better if there was update checking. Also, I do not mind being able to submit bug reports. As for the telemetry, I am willing to provide vague information (e.g. data that is not personally identifiable and does not contain specific data such as audio samples or filenames) especially if it helps developers to track bugs or determine the priority of issues.

Thank you for the continued development of Audacity. It has been a tough week, but I wish that people's trust will never be betrayed intentionally.

**Comment by mc776** [comment]
Bold of you to assume that anyone would ever trust anything you say ever again.

**Comment by Rexogamer** [comment]
Thank you for reconsidering things - especially regarding the under-13 usage discouragement. Hopefully things can settle and Audacity will continue to improve :)
